pkg/federation/sdlmerge/sdlmerge.go

```diff
@@ -93,3 +93,24 @@ func (m *normalizer) normalize(operation *ast.Document) error {
 
 	return nil
 }
+
+func AddQueryExtendKeyWord(doc *ast.Document) {
+	rootNodes := doc.RootNodes
+	for i, node := range rootNodes {
+		if node.Kind == ast.NodeKindObjectTypeDefinition {
+			switch string(node.NameBytes(doc)) {
+			case "Query", "Mutation", "Subscription":
```

Comment:
Does this also work in the case of e.g...?

```graphql
schema {
  query: RenamedQuery
}
```

There's a Index field on ast.Document which holds the index of all types as well as the root operation types.